addon/components/new-bourbon-select-field.js

```diff
@@ -119,32 +124,28 @@ export default Component.extend(ClickHandlerMixin, {
 
       let index = this.get('internalContent').indexOf(valueHolder);
 
+      if ((valueHolder && !valueHolder.enabled) || ((!this.get('prompt') && index === -1))) {
```

Comment:
I think the first `valueHolder` and `index === -1` are both checking for existence (or non-existence), right? Might want to be more consistent here, like maybe change the first `valueHolder` to `index !== -1`.

Reply:
that is a good call.  i can change that.

Reply:
so...`valueHolder` coming back `undefined` which is different than the `index !== 1` check.  and if i take it out then `!valueHolder.enabled` throws an error. so i will add `index !== 1` in addition to what i already have.